src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -86,6 +89,24 @@ protected static function bootKernel(array $options = [])
         return static::$kernel;
     }
 
+    /**
+     * Provides a dedicated test container with access to both public and private
```

Comment:
I think it's important to indicate that it does not include removed private services (e.g. those that are inlined or not used at all).